### PR TITLE
Add support for temporal v1.24.x

### PR DESCRIPTION
**Pull request by alexandrevilain**

### CODE REVIEW

File: docs/features/admin-tools.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ metadata:
...
+  version: 1.24.2
   numHistoryShards: 1
   # [...]
   admintools:
     enabled: true
     # You can specify the admin tools version if needed.
     # Check available tag you can check by the link below
     # https://hub.docker.com/r/temporalio/admin-tools/tags
-    version: 1.23.0
+    version: 1.24.1-tctl-1.18.1-cli-0.12.0
```
Comment (larhauga) [inline docs/features/admin-tools.md:22]: new admin-image to match matching temporal version (1.24.2)
```suggestion
    version: 1.24.2-tctl-1.18.1-cli-0.13.2
```

File: api/v1beta1/temporalcluster_defaults.go [hunk 2]
```diff
@@ -26,13 +26,14 @@ import (
...
+	defaultTemporalVersion = "1.24.2"
 	defaultTemporalImage   = "temporalio/server"
 
-	defaultTemporalUIVersion = "2.25.0"
+	defaultTemporalUIVersion = "2.27.3"
 	defaultTemporalUIImage   = "temporalio/ui"
 
-	defaultTemporalAdmintoolsImage = "temporalio/admin-tools"
+	defaultTemporalAdmintoolsImage   = "temporalio/admin-tools"
+	defaultTemporalAdmintoolsVersion = "1.24.1-tctl-1.18.1-cli-0.12.0"
```
Comment (larhauga) [inline api/v1beta1/temporalcluster_defaults.go:36]:
```suggestion
	defaultTemporalAdmintoolsVersion = "1.24.2-tctl-1.18.1-cli-0.13.2"
```

**Review by larhauga — COMMENTED** [review]
Nice work! 
I see there is a new version of the admin-tools image matching temporal version, which would be nice to update.

File: api/v1beta1/temporalcluster_defaults.go (hunk 2 above)
Comment (alexandrevilain) [reply api/v1beta1/temporalcluster_defaults.go]: Thanks :)

**Pull request merged**